src/js/components/LandingSection.tsx

```diff
@@ -32,7 +32,7 @@ const LandingSection = (): JSX.Element => {
 		<>
 			{/* 
 			//@ts-ignore */}
-			<MyLoopNavBar logo='/img/logo-in-pill.svg' backgroundColor='transparent' />
+			<MyLoopNavBar logo="/img/new-logo-with-text-white.svg" logoScroll="/img/new-logo-with-text.svg"/>
```

Comment:
This is fine as-is. FYI, /img/gl-logo gets you to a set of canonical logos, from the shared https://github.com/good-loop/icon repo

Reply:
Where is the gl-logo repo desginated to go - ~/winterwell/icon/gl-logo?